### ISSUE TITLE
Create a logo for DataFrame

### ISSUE THREAD

**Issue by olekscode** [PolyMathOrg/DataFrame#223]
I think that DataFrame is now mature enough to have its own logo :)

Check out the logos of:

- PolyMath: https://github.com/PolyMathOrg/PolyMath
- Cormas: https://github.com/cormas/cormas
- Pandas: https://pandas.pydata.org/
- Multiple projects under the NumFocus umbrella: https://numfocus.org/sponsored-projects

**Comment by SergeStinckwich** [comment]
I agree with that. Do you want to keep the DataFrame name or have a more fancy name?

**Comment by olekscode** [comment]
@SergeStinckwich I would love to have a more fancy name!
Because when we talk about this library, we often have to say `Pharo DataFrame`... which is much worse than `pandas` for example

Maybe we can all brainstorm to find some good name for Pharo data frame library

**Comment by exrol** [comment]
#280 Here is a pull request :-)

**Comment by SergeStinckwich** [comment]
Thank you!
